src/pods/canvas/canvas.vm.ts

```diff
@@ -36,3 +36,19 @@ export interface DatabaseSchemaVm {
   tables: TableVm[];
   relations: RelationVm[];
 }
+
+export interface Size {
+  width: number;
+  height: number;
+}
```

Comment:
I have updated _main_ branch, now Size is located in core/model or similar, could you please remove this _Size_ from here and use the one located in core/common? thanks

Reply:
It's fixed.